frontend/index.html

```diff
@@ -5,12 +5,12 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/x-icon" href="/src/favicon.ico" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Vite App</title>
+  <title>Tilup</title>
```

Comment:
```suggestion
  <title>Tilup 🌷</title>
```

Reply:
Good change. Now it looks ready to merge.

Reply:
Should be Tulip instead of Tilup, no?